1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,8 +16,7 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
-  /* todo */
+let thingsICanBuy = store.filter(obj => obj.price < money).map(obj => obj.item);
```

Comment:
great use of array methods. 
What happens if your money is exactly the price of the item?